src/qibocal/protocols/characterization/resonator_spectroscopy.py

```diff
@@ -192,9 +255,11 @@ def _update(results: ResonatorSpectroscopyResults, platform: Platform, qubit: Qu
     # therefore we update also the readout amplitude
     if len(results.bare_frequency) == 0:
         update.readout_amplitude(results.amplitude[qubit], platform, qubit)
+        if results.attenuation[qubit] is not None:
+            update.readout_attenuation(results.attenuation[qubit], platform, qubit)
     else:
         update.bare_resonator_frequency(results.bare_frequency[qubit], platform, qubit)
 
 
 resonator_spectroscopy = Routine(_acquisition, _fit, _plot, _update)
-"""ResonatorSpectroscopy Routine object."""
+"""ResonatorSpectroscopyAttenuation Routine object."""
```

Comment:
```suggestion
"""ResonatorSpectroscopy Routine object."""
```
At this point we can just call it `ResonatorSpectroscopy`.